src/rgw/rgw_sal_motr.cc

```diff
@@ -2591,7 +2698,7 @@ int MotrAtomicWriter::complete(size_t accounted_size, const std::string& etag,
   obj.meta.encode(bl);
   ldpp_dout(dpp, 20) <<__func__<< ": lid=0x" << std::hex << obj.meta.layout_id
                                                            << dendl;
-  if (info.versioning_enabled()) {
+  if (bkt_versioned) {
```

Comment:
why change needed, both should give pretty much same value. 
Also if we decide to change, then it may be needed at line 2678 also ?